dev-haskell/doctest/doctest-0.23.0.ebuild

```
@@ -15,11 +17,15 @@ LICENSE="MIT"
 SLOT="0/${PV}"
 KEYWORDS="~amd64"
 
+PATCHES=( "${FILESDIR}"/${P}-test-updates.patch
```

Just a future formatting tip:

```suggestion
PATCHES=(
    "${FILESDIR}"${P}-test-updates.patch
```

---

It doesn't work for me without the /

---

I think @solpeth meant

```suggestion
PATCHES=(
    "${FILESDIR}"/${P}-test-updates.patch
```